.phan/config.php

```diff
@@ -316,7 +316,7 @@
 
     // The number of processes to fork off during the analysis
     // phase.
-    'processes' => 1,
+    'processes' => 2,
```

Comment:
Was this changed to improve testing speed?

Reply:
it was just a leftover, I tested to see if I get some speed, but I didn't saw any improvement.